src/contrib/cluster/Akka.Cluster.Metrics/Serialization/EWMA.cs

```diff
@@ -0,0 +1,97 @@
+// //-----------------------------------------------------------------------
+// // <copyright file="EWMA.cs" company="Akka.NET Project">
+// //     Copyright (C) 2009-2019 Lightbend Inc. <http://www.lightbend.com>
+// //     Copyright (C) 2013-2019 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// // </copyright>
+// //-----------------------------------------------------------------------
+
+using System;
+
+namespace Akka.Cluster.Metrics.Serialization
+{
+    public sealed partial class NodeMetrics
+    {
+        public static partial class Types
+        {
+            /// <summary>
+            /// The exponentially weighted moving average (EWMA) approach captures short-term
+            /// movements in volatility for a conditional volatility forecasting model. By virtue
+            /// of its alpha, or decay factor, this provides a statistical streaming data model
+            /// that is exponentially biased towards newer entries.
+            /// 
+            /// http://en.wikipedia.org/wiki/Moving_average#Exponential_moving_average
+            /// 
+            /// An EWMA only needs the most recent forecast value to be kept, as opposed to a standard
+            /// moving average model.
+            /// 
+            /// Parameter 'alpha' decay factor, sets how quickly the exponential weighting decays for past data compared to new data,
+            ///   see http://en.wikipedia.org/wiki/Moving_average#Exponential_moving_average
+            /// 
+            /// Parameter 'value' the current exponentially weighted moving average, e.g. Y(n - 1), or,
+            ///             the sampled value resulting from the previous smoothing iteration.
+            ///             This value is always used as the previous EWMA to calculate the new EWMA.
+            /// </summary>
+            public sealed partial class EWMA 
```

Comment:
This class is `partial`, and it may require more efforts to use separate `EWMA` struct, because it is used by other `partial` classes... So if it is not a significant improvement, maybe better to keep it as it is - do you agree?